tests/make-test-app.sh

```diff
@@ -113,8 +113,8 @@ mkdir -p ${DIR}/files/share/icons/HighContrast/64x64/apps
 cp $(dirname $0)/org.test.Hello.png ${DIR}/files/share/icons/HighContrast/64x64/apps/${APP_ID}.png
 
 
-mkdir -p ${DIR}/files/share/appdata
-cat <<EOF > ${DIR}/files/share/appdata/${APP_ID}.xml
+mkdir -p ${DIR}/files/share/metainfo
+cat <<EOF > ${DIR}/files/share/metainfo/${APP_ID}.xml
```

Comment:
I think this test is unrealistic: the file is really `${ID}.metainfo.xml` or `${ID}.appdata.xml`, where `${ID}` is equal to or starts with the app ID.

For the test, we should probably mimic the current Steam app, which has two AppStream files, `files/share/appdata:
com.valvesoftware.Steam.appdata.xml` and `files/share/metainfo/com.valvesoftware.Steam.cmd.metainfo.xml`. That covers all three axes (`metainfo/` vs `appdata/`, `.metainfo.xml` vs `.appdata.xml`, and with/without a suffix (in this case `.cmd`) attached to the app-ID.

Reply:
I'm not sure whether `com.valvesoftware.Steam.cmd.metainfo.xml` is really used for anything, or even whether it's considered correct.

Reply:
Ahh great there's an MR for the rename already, thanks @mwleeds . That makes sense to me.